examples/cm_ros_node.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
What if we call this file Ex03_CarMaker_Ros_Bridge.py?

Reply:
yes sure feel free to make any changes